src/markdown-pages/instant-observability/build-a-quickstart/create-a-quickstart.mdx

```diff
@@ -0,0 +1,450 @@
+                  "query": "SELECT sum(fdb_create_errors), sum(fdb_read_errors), sum(fdb_update_errors), sum(fdb_delete_errors) FROM Metric SINCE 30 MINUTES AGO TIMESERIES"
+                }
+              ],
+              "yAxisLeft": {
+                "zero": true
+              }
+            },
+            "linkedEntityGuids": null
+          },
+          {
+            "visualization": {
+              "id": "viz.line"
+            },
+            "layout": {
+              "column": 9,
+              "row": 7,
+              "height": 3,
+              "width": 4
+            },
+            "title": "FlashDB keys",
+            "rawConfiguration": {
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 3014901,
+                  "query": "SELECT count(fdb_keys) FROM Metric SINCE 5 MINUTES AGO TIMESERIES"
+                }
+              ],
+              "yAxisLeft": {
+                "zero": true
+              }
+            },
+            "linkedEntityGuids": null
+          }
+        ]
+      }
+    ]
+  }
+> ```
+
+Next, create a screenshot of your dashboard and add it to _dashboards_ directory. Give it the same name as your JSON file. Your _dashboard_ directory should look similar to the following:
+
+![flashDB dashboard](../../../images/instant-observability/flashDB-dashboard.png)
+
+## Add alerts to quickstart
+
+Next, you add alerts to the quickstart. You're going to add existing static alerts to the quickstart. The _alerts_ directory contain the templates for static alerts.
+To help you populate your .yml alert files, you can use New Relic One's [Graphql API](https://api.newrelic.com/graphiql) to get the json object. Open the Graphql APi and select your account from the drop down.
+
+![graphiql interface](../../../images/instant-observability/graphiql.png)
+
+Building a query is simple in graphiql. Check the appropriate boxes to build the query in Graphql. Your query should be same as follows:
+
+>```json
+>{
+  actor {
+    account(id: {_your account id_}) {
+      alerts {
+        nrqlCondition(id: "{_nrql conditions's id_}") {
+          ... on AlertsNrqlStaticCondition {
+            id
+            name
+            nrql {
+              query
+            }
+          }
+          terms {
+            operator
+            priority
+            threshold
+            thresholdDuration
+            thresholdOccurrences
+          }
+        }
+      }
+    }
+  }
+}
+>
+
+Running the above query in Graphql will respond with the details of your alert condition that you can use to update your quickstart.
+
+![cache hit ratio in graphql](../../../images/instant-observability/cache-hit-ratio-graphql.png)
+
+For static condition, take the static-alert.yml file and start populating it using the information returned from above query. 
+First, name the file as **Cache Hit Ratio.yml** and populate it as follow:
```

Comment:
```suggestion
First, name the file _CacheHitRatio.yml_ and populate it:
```

A few suggestions here:

1. Remove some unnecessary words
2. Use italics for filenames
3. Don't have spaces in file names (you justified this already, but still a suggestion)

Reply:
I named the files with spaces to match the file name style for alerts in other quickstarts. elastic search quickstart named alerts as "Cluster health.yml", "Index Health.yml". @jaesius what's your opinion?